src/rsz/src/Resizer.cc

```diff
@@ -452,10 +452,10 @@ bool Resizer::canRemoveBuffer(Instance* buffer, bool honorDontTouchFixed)
       out_db_net->setDoNotTouch(false);
     }
   }
-  bool out_net_ports = hasPort(out_net);
+  bool out_net_ports = out_net && hasPort(out_net);
```

Comment:
Why is this necessary as is already handled in the method.  Same in two other places
```
bool Resizer::hasPort(const Net* net)
{
  if (!net) {
    return false;
  }
```